tests/analysis/Decompilation/dynamic_native/array.c

```diff
@@ -0,0 +1,27 @@
+#include <stdint.h>
+
+struct __attribute__((packed)) X {
+  uint16_t a[10];
+  uint64_t b;
+};
+
+uint16_t __attribute__((noinline)) getA(struct X *x) {
+  uint16_t max = 0;
+  for (int i = 0; i < 10; i++) {
+    uint16_t cur = x->a[i];
+    if (cur > max)
+      max = cur;
+  }
+
+  return max;
+}
+
+uint64_t __attribute__((noinline)) getB(struct X *x) { return (x->b) * 2; }
```

Comment:
Let's not have one line functions.
Also drop superfluous parenthesis.

Reply:
fixed